server/datastore/mysql/migrations/tables/20230303135738_AddMDMIdPAccountsTable.go

```diff
@@ -0,0 +1,27 @@
+package tables
+
+import (
+	"database/sql"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20230303135738, Down_20230303135738)
+}
+
+func Up_20230303135738(tx *sql.Tx) error {
+	_, err := tx.Exec(`
+    CREATE TABLE mdm_idp_accounts (
+      uuid         varchar(255) NOT NULL,
+      username     varchar(255) NOT NULL,
```

Comment:
Just a thought, any chance you'll query by `username` in this table (that is, `WHERE username = ?`)? If so it'd be good to have an index on it. If unsure, better leave it like that and add only if and when we need it.

Reply:
I think with the current implementation we would only query this table by `uuid`:

1. the enrollment profile `ServerURL` will have a `ref` query string containing the `uuid`
2. when the device enrolls, we fetch the IdP account by the`ref` value and send a `AccountConfiguration` MDM command to create the local account.

I think at this point we're unsure about fetching by `username`, so as you suggest I'll go ahead and merge. Thanks for double checking!